src/components/rooms/FeaturedRooms.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+import { RoomContext } from '../../context';
+import Loading from '../Loading';
+import { Room } from './index';
+import Title from '../Title';
+
+export default function FeaturedRooms() {
+  const context = useContext(RoomContext);
+
+  let { loading, featuredRooms: rooms } = context;
```

Comment:
```suggestion
  const { loading, featuredRooms: rooms } = context;
```
always const, no mutations